mantaray/backend.py

```diff
@@ -293,6 +300,12 @@ def _handle_received_message(self, msg: _ReceivedAndParsedMessage) -> None:
             self.event_queue.put(UserQuit(msg.sender, reason or None))
             return
 
+        if msg.command == "KICK":
+            assert msg.sender is not None
+            kicker = msg.sender
+            channel, kicked_nick, reason = msg.args
+            self.event_queue.put(Kick(kicker, channel, kicked_nick, reason or None))
```

Comment:
Put `return` after this, so that kick messages don't show up in the server view (`irc.libera.chat`).

Reply:
return caused the following errors to appear https://dpaste.com/CP4APZKHV

Might be that it's on a wrong place though.

Reply:
The error was elsewhere, return stays :)